### PR TITLE
Making more tests pass on travis

### PR DESCRIPTION
**Pull request by PeterKrcmar0**
This PR fixes the tests on Travis for:
- Favorites Fragment
- Track Properties Activity

It also removes the dummy tests.

---

The issue with the fragments was the _swipe down to refresh_ view. The refresh ran for too long while loading the data because of the slow emulator and travis doesn't like that. A (pretty ugly) fix was manually triggering the `loadData()` method of the fragment.

### CODE REVIEW

File: app/src/androidTest/java/ch/epfl/sweng/runpharaa/FavoritesFragmentTest.java [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+        sleep(1000);
+
+        onView(allOf(withId(R.id.emptyMessage), isDisplayed())).check(matches(withText(targetContext.getResources().getString(R.string.no_favorite))));
+    }
+
+    @Test
+    public void testFavoritesAppears() throws Throwable {
+        sleep(2000);
+        runOnUiThread(() ->((FragmentNearMe)mActivityRule.getActivity().getSupportFragmentManager().getFragments().get(0)).onRefresh());
+        /*onView(allOf(withId(R.id.cardListId), isDisplayingAtLeast(75))).perform(
```
Comment (AntonRagot) [inline app/src/androidTest/java/ch/epfl/sweng/runpharaa/FavoritesFragmentTest.java]: Just make sure to delete the comments if it does not work

File: app/src/androidTest/java/ch/epfl/sweng/runpharaa/TrackPropertiesActivityTest.java [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+    }
+
+    @Rule
+    public ActivityTestRule<TrackPropertiesActivity> mActivityRule =
+            new ActivityTestRule<>(TrackPropertiesActivity.class, false, false);
+
+    @Before
+    public void initUserAndTracks() {
+        User.instance = Database.getUser();
+        //User.set("FakeUser", 2000, Uri.parse(""), new LatLng(37.422, -122.084), "aa");
```
Comment (somecookie) [inline app/src/androidTest/java/ch/epfl/sweng/runpharaa/TrackPropertiesActivityTest.java]: Delete useless comments

File: app/src/androidTest/java/ch/epfl/sweng/runpharaa/TrackPropertiesActivityTest.java [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+    public void testCommentTooLong(){
+        Track t1 = createTrack();
+        launchWithExtras(t1);
+        sleep(5_000);
+        onView(withId(R.id.commentsID)).perform(click());
+
+        sleep(2000);
+        //onView(withId(R.id.comments_editText)).perform(click());
+
+        onView(withId(R.id.comments_editText)).perform(replaceText("Mais, vous savez, moi je ne crois pas " +
```
Comment (somecookie) [inline app/src/androidTest/java/ch/epfl/sweng/runpharaa/TrackPropertiesActivityTest.java]: magique

File: app/src/androidTest/java/ch/epfl/sweng/runpharaa/FavoritesFragmentTest.java (hunk 1 above)
Comment (PeterKrcmar0) [reply app/src/androidTest/java/ch/epfl/sweng/runpharaa/FavoritesFragmentTest.java]: I removed them in the latest commit, thanks for pointing it out.

File: app/src/androidTest/java/ch/epfl/sweng/runpharaa/TrackPropertiesActivityTest.java (hunk 2 above)
Comment (PeterKrcmar0) [reply app/src/androidTest/java/ch/epfl/sweng/runpharaa/TrackPropertiesActivityTest.java]: They're gone !

**Pull request merged**